dds_cli/data_lister.py

```diff
@@ -592,3 +612,43 @@ def __print_users_table(self, research_users):
                 dds_cli.utils.console.print(table)
         else:
             raise exceptions.NoDataError("No users found.")
+
+    # Project info listing
+    def __print_project_info_table(self, project_info):
+        # Column formatting
+        default_format = {"justify": "left", "style": "", "footer": "", "overflow": "fold"}
+        column_formatting = {
+            **{
+                x: default_format
+                for x in ["Project ID", "Created by", "Status", "Last updated", "Size"]
+            },
+        }
+        table = Table(
+            title="Project info",
+            show_header=True,
+            header_style="bold",
+            caption=(f"Information about project {project_info['Project ID']}"),
+        )
+        # Add columns to table
+        for colname, colformat in column_formatting.items():
+            table.add_column(
+                colname,
+                justify=colformat["justify"],
+                style=colformat["style"],
+                footer=colformat["footer"],
+                overflow=colformat["overflow"],
+            )
+
+        # Add all column values for the project row to table
+        new_row = []
+        for c in column_formatting:
+            new_row.append(
+                escape(
+                    dds_cli.utils.format_api_response(
+                        response=project_info[c], key=c, binary=self.binary
+                    )
+                )
+            )
+        table.add_row(*new_row)
+
+        dds_cli.utils.console.print(table)
```

Comment:
I believe this can be replaced with a call to `dds_cli.utils.create_table` (see comment in `show_project_info`)

I see that the same thing probably goes for `__print_users_table`, so we can improve on that one later at some point

```suggestion
```